lib/middleware/check-progress.js

```diff
@@ -18,7 +18,7 @@ module.exports = function checkProgress(route, controller, steps, start) {
         var potentialPaths = _.pluck(controller.options.forks, 'target')
             .concat(controller.options.next);
 
-        var nextStep = controller.getNextStep(req, res).replace(/\/edit$/, '');
+        var nextStep = controller._getForkTarget(req, res);
```

Comment:
only thing I would say is I would rather we weren't using underscored methods as public.